package.json

```diff
@@ -58,7 +59,7 @@
     "typescript": "3.4.1"
   },
   "peerDependencies": {
-    "@aws-cdk/aws-ecs": "^0.25.1",
-    "@aws-cdk/cdk": "^0.25.1"
+    "@aws-cdk/aws-ecs": "^0.25.1 || ^0.27.0",
+    "@aws-cdk/cdk": "^0.25.1 || ^0.27.0"
```

Comment:
btw, We should specify same version as devDependencies here?

Reply:
Yes, we should specify only `^0.27.0`, because the changes in this P-R contain incompatible with 0.25.1.